corehq/blobs/sql_templates/get_expired_blobs.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE FUNCTION get_expired_blobs(
```

Comment:
I don't see this being called anywhere

Reply:
Good catch. I had originally used it in the blob expiration code, but then refactored it later but forgot to remove this part. https://github.com/dimagi/commcare-hq/pull/21569/commits/afe5beaf12a7d5480d6f6ba90e78eec6d3502709